it/mail/chrome/communicator/wallet/wallet.properties

```diff
@@ -61,8 +61,8 @@ SaveTheseValuesEncrypted = Utilizza il Gestore password per ricordare questi dat
 SaveThisValueObscured = Utilizza il Gestore password per ricordare questo valore.
 SaveThisValueEncrypted = Utilizza il Gestore password per ricordare questo valore.
 SaveThisPasswordObscured= Utilizza il Gestore password per ricordare questa password.
-SaveThisPasswordEncrypted = Utilizza Gestione password per ricordare questa password.
-xSaveTheseValuesObscured = Ricorda questi dati. I dati archiviati non sono protetti da password.
+SaveThisPasswordEncrypted = Utilizzare Gestione password per ricordare questa password.
```

Comment:
CORRECTIONS NEEDED. 

In order to improve the consistency of the text, please use "Utilizza" instead of "Utilizzare"